hokusai/production.yml

```diff
@@ -173,9 +215,15 @@ spec:
             fieldRef:
               fieldPath: metadata.labels['app.kubernetes.io/version']
         envFrom:
+        - configMapRef:
+            name: secrets-config
         - configMapRef:
             name: artsy-unleash-proxy-environment
-        image: unleashorg/unleash-proxy
+        volumeMounts:
+        - name: secrets
+          mountPath: /secrets
+          readOnly: true
+        image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/artsy-unleash-proxy:latest
```

Comment:
Looking for some advice on whether this is the way we would like to go. I'd prefer to use the [artsy-unleash](https://us-east-1.console.aws.amazon.com/ecr/repositories/private/585031190124/artsy-unleash?region=us-east-1) ecr repo but im not sure if thats going to complicate things. Maybe its more straight forward to do this?

Reply:
i think Hokusai currently maps a Github repo with an ECR repo, one to one. so we would have to use a separate ECR repo.

the problem then becomes, how do we get CI to build two Docker images.

Reply:
oh i see, the suggestion is to build proxy image [locally](scripts/build_push_proxy.sh).

Reply:
> the problem then becomes, how do we get CI to build two Docker images.

Right, I couldnt think of one or find an example of us already doing that so reached for the manual script...